### PR TITLE
CIV-8903 dashboard settlement agreement

### PR DESCRIPTION
**Pull request by paul-pearson**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-8903

### Change description ###

Implement new statuses for settlement agreement.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/common/models/dashboard/dashboardItem.ts [hunk 1]
```diff
@@ -78,6 +78,14 @@ export class DashboardClaimantItem extends DashboardItem {
       WAITING_FOR_CLAIMANT_TO_RESPOND: {translationKey: 'PAGES.DASHBOARD.STATUS_CLAIMANT.NOT_ADMITTED_CLAIMANT'},
       WAITING_COURT_REVIEW: { translationKey: 'PAGES.DASHBOARD.STATUS_CLAIMANT.WAITING_COURT_REVIEW'},
       DEFAULT_JUDGEMENT: {translationKey: 'PAGES.DASHBOARD.STATUS_CLAIMANT.CLAIMANT_REQUESTED_CCJ'},
+      CLAIMANT_SEND_SETTLEMENT_AGREEMENT: {translationKey: 'PAGES.DASHBOARD.STATUS_CLAIMANT.CLAIMANT_SIGNED_SETTLEMENT_AGREEMENT'},
```
Comment (manish14836) [inline src/main/common/models/dashboard/dashboardItem.ts:81]: CLAIMANT_SEND_SETTLEMENT_AGREEMENT should it be CLAIMANT_SIGNED_SETTLEMENT_AGREEMENT? 
same for CLAIMANT_SEND_SETTLEMENT_AGREEMENT_DEADLINE_EXPIRED

File: src/main/common/models/dashboard/dashboardItem.ts [hunk 2]
```diff
@@ -180,6 +188,9 @@ export class DashboardDefendantItem extends DashboardItem {
         parameter: [paramClaimantName, paramAdmittedAmount],
       },
       SDO_ORDER_CREATED: { translationKey: 'PAGES.DASHBOARD.STATUS_DEFENDANT.SDO_ORDER_STATUS' },
+      CLAIMANT_AND_DEFENDANT_SIGNED_SETTLEMENT_AGREEMENT: {translationKey: 'PAGES.DASHBOARD.STATUS_DEFENDANT.BOTH_SIGNED_SETTLEMENT_AGREEMENT' },
+      CLAIMANT_SEND_SETTLEMENT_AGREEMENT: {translationKey: 'PAGES.DASHBOARD.STATUS_DEFENDANT.CLAIMANT_SIGNED_SETTLEMENT_AGREEMENT', parameter: [paramClaimantName] },
```
Comment (manish14836) [inline src/main/common/models/dashboard/dashboardItem.ts:192]: same here

**Review by manish14836 — APPROVED** [review]

**Review by jeswanth-hmcts — APPROVED** [review]

File: src/main/modules/i18n/locales/cy.json [hunk 3]
```diff
@@ -1281,7 +1285,10 @@
         "SETTLEMENT_SIGNED": "Mae'r ddau ohonoch wedi llofnodi cytundeb setlo.",
         "SDO_ORDER_STATUS": "Mae’r llys wedi cyhoeddi gorchymyn; ni ellir parhau â’r hawliad hwn ar-lein a bydd yn rhaid defnyddio ffurflenni papur.",
         "TRIAL_OR_HEARING_SCHEDULED": "Mae Gwrandawiad neu Dreial wedi'i drefnu ar gyfer yr achos hwn",
-        "WAITING_COURT_REVIEW": "Aros i'r llys adolygu'r achos"
+        "WAITING_COURT_REVIEW": "Aros i'r llys adolygu'r achos",
```
Comment (portizdev) [inline src/main/modules/i18n/locales/cy.json:1288]: Pls keep the translations in alphabetical order

File: src/main/modules/i18n/locales/en.json [hunk 4]
```diff
@@ -1242,6 +1242,10 @@
         "REQUESTED_MORE_TIME_TO_RESPOND": "{{defendantName}} has requested more time to respond.",
         "RESPONSE_BY_POST": "The claim will continue by post",
         "WAITING_COURT_REVIEW": "Wait for the court to review the case",
+        "BOTH_SIGNED_SETTLEMENT_AGREEMENT": "You’ve both signed a settlement agreement.",
+        "DEFENDANT_REJECTED_SETTLEMENT_AGREEMENT": "{{ defendantName }} has rejected your settlement agreement. You can request a County Court Judgment against them.",
```
Comment (portizdev) [inline src/main/modules/i18n/locales/en.json:1246]: Pls keep the translations in alphabetical order

File: src/main/modules/i18n/locales/en.json [hunk 5]
```diff
@@ -1281,7 +1285,10 @@
         "SETTLEMENT_SIGNED": "You’ve both signed a settlement agreement.",
         "SDO_ORDER_STATUS": "An order has been issued by the court, this claim will now proceed offline",
         "TRIAL_OR_HEARING_SCHEDULED": "A Trial or Hearing has been scheduled for this case",
-        "WAITING_COURT_REVIEW": "Wait for the court to review the case"
+        "WAITING_COURT_REVIEW": "Wait for the court to review the case",
+        "BOTH_SIGNED_SETTLEMENT_AGREEMENT": "You’ve both signed a settlement agreement.",
```
Comment (portizdev) [inline src/main/modules/i18n/locales/en.json:1289]: Pls keep the translations in alphabetical order

**Review by neeta-hmcts — APPROVED** [review]

**Pull request merged**